deployment_planning_guide/_topics/capacity.md

```diff
@@ -58,28 +58,28 @@ database (ovirt\_engine\_history) is located. Usually, this is the RHV-M server.
 
     <div class="important">
 
-    For RHV 4.2, the PostgreSQL database is delivered as a software collection in version 9.5 and must be enabled first. Therefore, to run the following psql commands, you need to enable the *rh-postgresql95* collection and load into the current shell prompt by using the source command:
+    For RHV 4.2 and 4.3, the PostgreSQL database is delivered as a software collection in version 9.5 and must be enabled first. Therefore, to run the following psql commands, you need to enable the *rh-postgresql95* collection and load into the current shell prompt by using the source command:
 
     $ source /opt/rh/rh-postgresql95/enable
 
     </div>
 
 3. Create the user for {{ site.data.product.title_short }} to be granted read-only access to the history database:
 
-        $ psql -U postgres -c "CREATE ROLE cfme WITH LOGIN ENCRYPTED PASSWORD '[password]';" -d ovirt_engine_history
+        $ psql -U postgres -c "CREATE ROLE manageiq WITH LOGIN ENCRYPTED PASSWORD '[password]';" -d ovirt_engine_history
```

Comment:
This role name can be anything, right?  wondering if we should productize this.

Reply:
Or something random like "example_role_name"

Reply:
yeah true, would `title_short_l` work downstream as well?

Reply:
Alright update to `example_role_name`